### PR TITLE
Tasks 1437-1442 : DoQA tasks

### PR DESCRIPTION
**Pull request by kuntalkumarpal**

### CODE REVIEW

File: tasks/task1437_doqa_cooking_question_generation.json [hunk 1]
```diff
@@ -0,0 +1,5525 @@
...
+	"Input_language": [
+		"English"
+	],
+	"Output_language": [
+		"English"
+	],
+	"Instruction_language": [
+		"English"
+	],
+	"Categories": [
```
Comment (yeganehkordi) [inline tasks/task1437_doqa_cooking_question_generation.json]: Please update the categories and add the domain field based on the task-hierarchy.

File: tasks/task1437_doqa_cooking_question_generation.json [hunk 2]
```diff
@@ -0,0 +1,5525 @@
...
+		},
+		{
+			"input": "CONTEXT: It not huge, it's just the difference from freezer to room temperature you are worrying aboutE.g. -20\u00b0C to 20\u00b0C, is A 40\u00b0C shift. The shift was going to be 20\u00b0C to 100\u00b0C anyway. There is no physical reasons why this would be anymore stressfulFrom room temperature you are raising it 80\u00b0C, from frozen you are raising it 120\u00b0C. Not a problem in the normal temperature range for glassThe freezing temperature of water (most common item in food) has no relation to the freezing temperature of glass etc Pyrex and other glasses can be damaged if one part of them is instantly heated or cooled by a 100\u00b0C or so Q: Is it safe to microwave Pyrex containers immediately after removing them from the freezer and removing the plastic lid? A: Pyrex and other glasses can be damaged if one part of them is instantly heated or cooled by a 100\u00c2\u00b0C or so",
```
Comment (yeganehkordi) [inline tasks/task1437_doqa_cooking_question_generation.json]: - Make sure that your json is human readable (use proper indentation; e.g., in Python: json.dumps(your_json_string, indent=4, ensure_ascii=False))
- Please use `<sep>` token for separating context and conversations.

File: tasks/task1437_doqa_cooking_question_generation.json [hunk 3]
```diff
@@ -0,0 +1,5525 @@
...
+			"output": [
+				""
```
Comment (yeganehkordi) [inline tasks/task1437_doqa_cooking_question_generation.json]: Please drop the instances without output.

File: tasks/task1438_doqa_cooking_answer_generation.json [hunk 4]
```diff
@@ -0,0 +1,1757 @@
...
+	"Output_language": [
+		"English"
+	],
+	"Instruction_language": [
+		"English"
+	],
+	"Categories": [
+		"Question Generation"
+	],
+	"Definition": "Given a paragraph about cooking, and a set of conversational question answers about the paragraph, answer a follow-up question from the paragraph. The paragraph has a prefix 'CONTEXT:'. Each conversation question has a prefix Q: followed by answer prefix A: ",
```
Comment (yeganehkordi) [inline tasks/task1438_doqa_cooking_answer_generation.json]: Maybe change the prefix of the follow-up question and separate it with `<sep>`.

File: tasks/task1438_doqa_cooking_answer_generation.json [hunk 5]
```diff
@@ -0,0 +1,1757 @@
...
+			"output": "Traditionally (in Ireland) you wouldn't have a sauce with it but a parsely sauce",
+			"explanation": "The answer to the question is directly mentioned in the passage (parsely sauce)"
```
Comment (yeganehkordi) [inline tasks/task1438_doqa_cooking_answer_generation.json]: Please add a period at the end of the sentence (same for the second negative example).

**Comment by swarooprm** [comment]
@kuntalkumarpal, can you address reviewer comments?

**Comment by swarooprm** [comment]
> @kuntalkumarpal, can you address reviewer comments?

@yeganehkordi, check if you are happy or have any other feedback?

File: tasks/task1437_doqa_cooking_question_generation.json (hunk 2 above)
Comment (yeganehkordi) [reply tasks/task1437_doqa_cooking_question_generation.json]: Please address these comments.

File: tasks/task1437_doqa_cooking_question_generation.json (hunk 3 above)
Comment (yeganehkordi) [reply tasks/task1437_doqa_cooking_question_generation.json]: Please remove the instances without output.

File: tasks/task1438_doqa_cooking_answer_generation.json (hunk 4 above)
Comment (yeganehkordi) [reply tasks/task1438_doqa_cooking_answer_generation.json]: Please change the prefix of the follow-up question. It's better to differ from other questions.

File: tasks/task1438_doqa_cooking_answer_generation.json (hunk 5 above)
Comment (yeganehkordi) [reply tasks/task1438_doqa_cooking_answer_generation.json]: Please address this comment as well.

**Comment by kuntalkumarpal** [comment]
@yeganehkordi I was working on the fixes you suggested. You can now review and let me know if you find anything incorrect.

- Added <sep> between context and conversations and also before followup question
- Added new prefix for followup questions
- Added period in examples
- Indentation fixed
- Blank records dropped
- Added domain and updated categories from values present in taskhierarchy.md

**Comment by yeganehkordi** [comment]
Thanks! Looks good to me!
@kuntalkumarpal could you update the readme file based on the new categories?

**Pull request merged**